Sweden national page (#150)
elixir-europe/infectious-diseases-toolkit: national-resources/sweden.md

## Health authorities
<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies --->

There are multiple authorities in Sweden that determine public health policy and the measures that should be taken to protect public health. All are government authorities with different responsibilities.
- **[The Swedish Medical Products Agency (Läkemedelsverket)](https://www.lakemedelsverket.se/en)** is responsible for the approval of prescription medications and vaccinations, as well as for monitoring their efficacy and safety after approval. It also approves and controls clinical trials.

> - **[The Public Health Agency of Sweden (Folkhälsomyndigheten, FoHM)](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/)** is responsible for monitoring public health issues in Sweden. It collects and aggregates [statistics related to the effect of infectious diseases on public health](https://www.folkhalsomyndigheten.se/folkhalsorapportering-statistik/statistik-a-o/sjukdomsstatistik/) (e.g. the number of cases and hospitalisations related to infection with a given pathogen).

Author: It is basically a 'root directory' for all of the statistics that the public health agency collects on each disease, so you can find your pathogen in the list there and see all the statistics about it. So, I agree, that link itself is not a data source, but it is the source of all data sources they have. Maybe we just need to clarify that in text?

> - **[The Public Health Agency of Sweden (Folkhälsomyndigheten, FoHM)](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/)** is responsible for monitoring public health issues in Sweden. It collects and aggregates [statistics related to the effect of infectious diseases on public health](https://www.folkhalsomyndigheten.se/folkhalsorapportering-statistik/statistik-a-o/sjukdomsstatistik/) (e.g. the number of cases and hospitalisations related to infection with a given pathogen).

Reviewer: It looks useful in my opinion to have it clear in the text that that link will lead one to the source of many sources. Maybe add it to data sources and clarify?

> - **[The Public Health Agency of Sweden (Folkhälsomyndigheten, FoHM)](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/)** is responsible for monitoring public health issues in Sweden. It collects and aggregates [statistics related to the effect of infectious diseases on public health](https://www.folkhalsomyndigheten.se/folkhalsorapportering-statistik/statistik-a-o/sjukdomsstatistik/) (e.g. the number of cases and hospitalisations related to infection with a given pathogen).

Author: Clarified here, will also add to data sources

- **[The National Board for Health and  Welfare (Socialstyrelsen)](https://www.socialstyrelsen.se/en/)** is responsible for issuing guidelines and regulations related to public health. It also maintains multiple [clinical registers](https://www.socialstyrelsen.se/en/statistics-and-data/registers/) and [statistics on COVID-19](https://www.socialstyrelsen.se/en/statistics-and-data/statistics/statistics-on-covid-19/).

## Disease surveillance initiatives

> Multiple initiatives have been established for disease surveillance in Sweden. Below we give examples of ongoing initiatives. These initiatives generally monitor a broad range of pathogens.

Author: re-edited to make a slight change here

### Wastewater surveillance

Wastewater surveillance has proven to be an effective means of predicting outbreaks, with peaks in viral genome content shown to precede increases in cases and hopitalisations. This is also true of Swedish wastewater surveillance efforts, with  e.g. [Wang *et al.* (2022)](https://pubmed.ncbi.nlm.nih.gov/36035197/) showing that peaks in viral genome content preceded increases in cases by two weeks.

Multiple research groups across Sweden have monitored viral genome content in wastewater, including groups at the [University of Gothenburg](https://www.gu.se/en), [KTH Royal Institute of Technology](https://www.kth.se/en), and [Swedish University of Agricultural Sciences (SLU)](https://www.slu.se/en/). Throughout the COVID-19 pandemic these efforts largely focused on monitoring SARS-CoV-2, but the scope has expanded to include other pathogens. More information on the work by these groups is on the [Swedish COVID-19 and Pandemic Preparedness Portal](https://www.covid19dataportal.se/dashboards/wastewater/).

### Sentinel monitoring in healthcare

> The Swedish Public Health Agency coordinates [sentinel monitoring (information only available in Swedish)](https://www.folkhalsomyndigheten.se/mikrobiologi-laboratorieanalyser/mikrobiella-och-immunologiska-overvakningsprogram/sentinelovervakning/) for COVID-19 and influenza. In sentinel monitoring, samples are taken from patients with symptoms of influenza or acute respiratory infections. These samples are analysed by [The Swedish Public Health Agency](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/). The data is then used to inform national policy decisions, and is reported to the public, before being subsequently reported to the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en). ECDC then reports the date to the [World Health Organisation (WHO)](https://www.who.int/). 

Author: I have added links to the latest weekly reports that they do. Their reports to the public are in the main sentinel monitoring part that they do, linked under 'Hur presenteras data?' ('How is the data presented?') at the bottom of that page. I'm a little torn as to whether to point users to that section or whether to add the links to the reports (more reports could be added in future, the links I've included now might break.). 

Maybe it'd instead be better to change it so that it says something like ...

'The data is then used to inform national policy decisions, and is presented to the public in regular reports (see the 'Hur presenteras data?' (In English: 'How is the data presented?') section of the [sentinel monitoring page (information only available in Swedish)](https://www.folkhalsomyndigheten.se/mikrobiologi-laboratorieanalyser/mikrobiella-och-immunologiska-overvakningsprogram/sentinelovervakning/) to access the public reports. The data is subsequently reported to the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en), who then reports the data to the [World Health Organisation (WHO)](https://www.who.int/). '

> The Swedish Public Health Agency coordinates [sentinel monitoring (information only available in Swedish)](https://www.folkhalsomyndigheten.se/mikrobiologi-laboratorieanalyser/mikrobiella-och-immunologiska-overvakningsprogram/sentinelovervakning/) for COVID-19 and influenza. In sentinel monitoring, samples are taken from patients with symptoms of influenza or acute respiratory infections. These samples are analysed by [The Swedish Public Health Agency](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/). The data is then used to inform national policy decisions, and is reported to the public, before being subsequently reported to the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en). ECDC then reports the date to the [World Health Organisation (WHO)](https://www.who.int/). 

Reviewer: I think the comment suggestion is rather good. Can we go with that instead of what is in the commit?

> The Swedish Public Health Agency coordinates [sentinel monitoring (information only available in Swedish)](https://www.folkhalsomyndigheten.se/mikrobiologi-laboratorieanalyser/mikrobiella-och-immunologiska-overvakningsprogram/sentinelovervakning/) for COVID-19 and influenza. In sentinel monitoring, samples are taken from patients with symptoms of influenza or acute respiratory infections. These samples are analysed by [The Swedish Public Health Agency](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/). The data is then used to inform national policy decisions, and is reported to the public, before being subsequently reported to the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en). ECDC then reports the date to the [World Health Organisation (WHO)](https://www.who.int/). 

Author: Updated

## Dashboards and visualisation platforms

> 

Author: We debated this at the contentathon too, the decision across the national pages was to add individual links to each dashboard, so that's why we've left it like this. However, I am happy to change it to be more concise. Perhaps this is a rule that we need to come to at an editor level? Like 'if there are multiple pages all related to the same website then they should be presented as '...' ?

- [The Swedish Public Health Agency’s information on COVID-19 PCR tests ](https://www.folkhalsomyndigheten.se/PCR-covid-19/): shows the number of PCR tests performed over time in Sweden (only available in Swedish).  
- [The National Board for Health and  Welfare’s dashboard on COVID-19 statistics](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/): shows information on the number of COVID-19 cases, and survival rates, as well as information on post COVID-19 condition/longcovid both for different regions and over time (only available in Swedish).
- [The National Board for Health and  Welfare’s dashboard on data indirectly related to COVID-19](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/statistik-relaterad-till-covid-19/): shows information related to the number of influenza cases over time, as well as general mortality rates in Sweden (only available in Swedish and no longer updated as of January 2022).

> ## National data sources

Author: Would something like the below example be sufficient? If so, I'll change the others accordingly too, I just wanted to be sure I understood first.

> ## National data sources

Reviewer: Yes!
I will commit that one. Can you please add to the others?

<!--- A section to list and provide context to national data sources. Data source as defined in BY-COVID - those are online resources in which data can be obtained. --->

> - [Resource showing all of the available data from COVID-19 publications involving at least one individual affiliated with a Swedish institution](https://www.covid19dataportal.se/datasets/all/) (data is openly available).

Author: ```suggestion
- [Database of Swedish COVID-19 publications that make data available](https://www.covid19dataportal.se/datasets/all/). This resource lists all publications on COVID-19 that involve at least one researcher affiliated with a Swedish research institution and make their data openly available. The data has been categorised according to data type (e.g. social and humanities data, genomics data) to aid users (data is openly available).
```

> - [COVID-19 data sources from The Public Health Agency of Sweden](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/) (data is openly available).

Author: ```suggestion
- [Data on infectious diseases from The Public Health Agency of Sweden](https://www.folkhalsomyndigheten.se/folkhalsorapportering-statistik/statistik-a-o/sjukdomsstatistik/) (data is openly available). This resource is a directory of statistics (e.g. the number of cases detected, and the number of resulting hospitalisations) collected by the Public Health Agency of Sweden on multiple diseases. The statistics for each pathogen/disease can be accessed by locating a given pathogen in the directory (sorted alphabetically).
```

- [COVID-19 data sources from The National Board for Health and  Welfare](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/) (data is openly available).

> - [Infectious diseases data sources published on the SciLifeLab Data Repository](https://figshare.scilifelab.se/categories/Biomedical_and_clinical_sciences/24457?categories=24508) (some data is openly available, otherwise access to data is restricted).

Author: ```suggestion
- [Infectious diseases data sources published on the SciLifeLab Data Repository](https://figshare.scilifelab.se/categories/Biomedical_and_clinical_sciences/24457?categories=24508) (some data is openly available, otherwise access to data is restricted). The SciLifeLab Data Repository includes multiple different types of data (e.g. image data), the information found at the link shows only the data related to infectious disease, regardless of data type. 
```

> - [National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) (access to data is restricted). 

Author: ```suggestion
- [National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) (access to data is restricted). This resource includes four different types of data about patients; basic patient data (e.g. age), geographic data (e.g. hospital department), administrative data (e.g. data of. in/our patient check in), and medical data (e.g. main diagnosis).
```

> - [National Prescribed Drug Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-prescribed-drug-register/) (access to data is restricted). 

Author: ```suggestion
- [National Prescribed Drug Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-prescribed-drug-register/) (access to data is restricted). This resource provides official statistics related to drugs prescribed in Sweden. It is used to increase patient safety in relation to prescribed drugs.
```

> - [Quality Registry for SARS-CoV-2](https://covid19register.se/) (only available in Swedish) (access to data is restricted). 

Author: ```suggestion
- [Quality Registry for SARS-CoV-2 data](https://covid19register.se/) (only available in Swedish) (access to data is restricted). This resource provides quality indicators related to SARS-CoV-2 (e.g. the number or PCR tests done overall, and percentage of the population tested).

> - [National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) (access to data is restricted). This resource includes four different types of data about patients; basic patient data (e.g. age), geographic data (e.g. hospital department), administrative data (e.g. data of. in/our patient check in), and medical data (e.g. main diagnosis).

Reviewer: ```suggestion
- [National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) (access to data is restricted). This resource includes four different types of data about patients; basic patient data (e.g. age), geographic data (e.g. hospital department), administrative data (e.g. data of inpatient check in), and medical data (e.g. main diagnosis).
```
typo correction

---
title: Sweden

> search_exclude: true <!---leave as “true” until page is ready to be made public--->

Reviewer 2: Don't forget to take this one out!

Author: Removed search_exclude now. Thank you, let me know if it should instead be set to 'false'
